ERCS/erc-7390.md

```diff
@@ -0,0 +1,366 @@
+---
+eip: 7390
+title: Vanilla Options
+description: An Interface for Vanilla Options
+author: Ewan Humbert (@Xeway) <xeway@protonmail.com>, Lassi Maksimainen (@mlalma) <lassi.maksimainen@gmail.com>
+discussions-to: https://ethereum-magicians.org/t/erc-7390-vanilla-option-standard/15206
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-09-02
+requires: 20
+---
+
+## Abstract
+
+This ERC defines a comprehensive set of functions and events facilitating seamless interactions (creation, management, exercising, etc.) for vanilla options. This standard ensures that options contracts conform to a common interface, facilitating the development of robust options trading platforms and enabling interoperability between dApps and protocols.
```

Comment:
Your abstract should give a high-level technical overview so that potential readers get an understanding of both what the proposal is *and* how it works.

I'd also define what a "vanilla option" is. My background is more on the software side, I assumed it was some sort of configuration system, while this proposal is more financial in nature.

Also, all ERCs promote interoperability, so no need to mention that in the abstract.

```suggestion
This ERC defines a comprehensive set of functions and events facilitating seamless interactions (creation, management, exercising, etc.) for vanilla options. It facilitates the development of robust options trading platforms.
```